MemeMe/memeStruct.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  memeStruct.swift
+//  MemeMe
+//
+//  Created by Waylon Kumpe on 7/14/22.
+//
+
+// MARK: - Import
+import Foundation
+import UIKit
+
+// MARK: - Meme.
```

Comment:
[MARK: -] Is reserved to divide your code into sections. This should just be [MARK: ]